sync-core/src/main/java/com/cloudant/sync/datastore/migrations/MigrateDatabase6To7.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2015 IBM Cloudant, Inc. All rights reserved.
```

Comment:
With the copyrights I'm pretty sure we can't use `IBM Cloudant, Inc.` since IBM Cloudant isn't incorporated